src/unstract/adapters/base.py

```diff
@@ -48,21 +49,26 @@ def get_json_schema() -> str:
     def get_adapter_type() -> AdapterTypes:
         return ""
 
-    def get_llm_instance(self, llm_config: dict[str, Any]) -> Optional[LLM]:
+    def get_llm_instance(self, llm_config: dict[str, Any]) -> LLM:
         # Overriding implementations use llm_config
-        return None
+        return MockLLM()
 
     def get_vector_db_instance(
         self, vector_db_config: dict[str, Any]
-    ) -> Union[BasePydanticVectorStore, VectorStore, None]:
+    ) -> Union[BasePydanticVectorStore, VectorStore]:
         # Overriding implementations use vector_db_config
-        return None
+        return SimpleVectorStore()
 
     def get_embedding_instance(
         self, embed_config: dict[str, Any]
-    ) -> Optional[BaseEmbedding]:
+    ) -> BaseEmbedding:
+        # TODO: Return mock embedding, Rempve type ignore
+        # Non-matching return type here to avoid the below error
+        # RuntimeError: There is no current event loop in
+        # thread 'ThreadPoolExecutor-0_0'
+
         # Overriding implementations use embed_config
-        return None
+        return  # type: ignore
```

Comment:
@gaya3-zipstack Since this issue is fixed in llama-index can we return `MockEmbedding` here?

Reply:
Done.